maps/src/Map.tsx

```diff
@@ -61,11 +74,11 @@ function Markers({ filteredData, selfLocation }: { filteredData: MarkerData["map
           fillOpacity={1}
           radius={radius / 4}
           opacity={0.75}
-        >
+          >
           <Popup>Sizin Konumunuz</Popup>
         </CircleMarker>
       }
-    </>
+  </>
```

Comment:
I think we may not have this in a cluster since it is only one location and people might want to see it.